ui/core/worker_pool.ts

```diff
@@ -151,21 +158,44 @@ export class WorkerPool {
 		return RaidSimResult.fromBinary(result);
 	}
 
+	// TODO: target specific workers?
+	async abortById(id: string): Promise<AbortResponse> {
+		const wait: Promise<Uint8Array>[] = [];
+		for (const worker of this.workers) {
+			wait.push(worker.doApiCall(SimRequest.abortById, AbortRequest.toBinary(AbortRequest.create({id})), ''))
+		}
+		const results = await Promise.all(wait);
```

Comment:
I know you dislike it, but you can just use:
```
const results = await Promise.all(this.workers.map(worker => .doApiCall(SimRequest.abortById, AbortRequest.toBinary(AbortRequest.create({id})), '')))
```

Reply:
I'd be fine with it in this case :D